src/main/java/uk/gov/hmcts/reform/notificationservice/data/NotificationStatus.java

```diff
@@ -3,4 +3,6 @@
 enum NotificationStatus {
 
     PENDING,
+    FAILED,
+    SENT
```

Comment:
are we using this ?

Reply:
not yet but it is part of the task

Reply:
if not yet, could've been added later when it is